js/data.js

```diff
@@ -1 +1,15 @@
 /* exported data */
+// function getMovies() {
+//   var xhr = new XMLHttpRequest();
+//   xhr.open('GET', 'https://imdb-api.com/en/API/Top250Movies/k_671i2qju');
+//   xhr.responseType = 'json';
+//   xhr.addEventListener('load', function () {
+//     console.log('xhr.response: ', xhr.response);
+//     console.log('xhr.response.items: ', xhr.response.items);
+//     console.log('xhr.response.items[0]: ', xhr.response.items[0]);
+//     console.log('xhr.response.items[0].crew: ', xhr.response.items[0].crew);
+//   });
+//   xhr.send();
+//   // return xhr.response.items;
+// }
+// getMovies();
```

Comment:
Please remove commented code prior to submitting for review

Reply:
<img width="725" alt="Screen Shot 2022-08-02 at 11 41 19 AM" src="https://user-images.githubusercontent.com/106217355/182449600-f6b202ae-b7e9-4cc4-a0f5-f9b4eb129ff2.png">